src/Test/Utilities/Portable/TestUtilities.csproj

```diff
@@ -9,7 +9,7 @@
     <AssemblyName>Roslyn.Test.Utilities</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Nonshipping>true</Nonshipping>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFrameworks>netstandard1.3;$(RoslynPortableTargetFrameworks)</TargetFrameworks>
```

Comment:
What's going on here? We build for `netstandard1.3`, `netcoreapp`, and `net46`? Shouldn't it just be `netstandard1.3` and we can move the System.Runtime.Loader reference into the `TestUtilities.CoreCLR` project?